src/components/ConfigSlider/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import Slider from '@react-native-community/slider';
+
+import {
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+} from '../Globals/ConfigMetric';
+
+import {
+  Button,
+  Modal,
+  ModalContent,
+  ModalWrapper,
+  ButtonText,
+  ModalButton,
+  ModalButtonWrapper,
+  PercentageDisplay,
+  ModalOpenButton
+} from './styles';
+import { View } from 'react-native';
+
+const ConfigSlider = () => {
```

Comment:
In our case, we have a range of values to show - `20-30, 30-40, 40-50, 50-60, 60-70, 70-80, 80-90, 90-100`. You are only showing an absolute value, so you will need to show these ranges on each slider option rather than a single value. So see if you can display `20-30` etc.